SlackChatbot/utils/trello_utils.py

```diff
@@ -0,0 +1,46 @@
+#placeholder trello code 
+def create_trello_card(board_name, list_name, card_name, card_desc=""):
+    # Placeholder logic for testing
```

Comment:
This placeholder logic should be replaced with actual Trello API integration or removed if not necessary.